tests/user_restricted/data/pds_scenarios.py

```diff
@@ -1,5 +1,5 @@
 retrieve = [
-    {"scenario":"retrieve_patient","patient":"5900038181", "response":{"address":[{"id":"9F4FF65D","line":["testing","testing","testing","testing","testing"],"period":{"start":"2021-01-22"},"postalCode":"HL12 6HL","use":"home"}],"birthDate":"2010-01-22","extension":[{"extension":[{"url":"language","valueCodeableConcept":{"coding":[{"code":"hy","display":"Armenian","system":"https://fhir.nhs.uk/R4/CodeSystem/UKCore-HumanLanguage","version":"1.0.0"}]}},{"url":"interpreterRequired","valueBoolean":False}],"url":"https://fhir.nhs.uk/R4/StructureDefinition/Extension-UKCore-NHSCommunication"}],"gender":"female","id":"5900038181","identifier":[{"extension":[{"url":"https://fhir.nhs.uk/R4/StructureDefinition/Extension-UKCore-NHSNumberVerificationStatus","valueCodeableConcept":{"coding":[{"code":"01","display":"Number present and verified","system":"https://fhir.nhs.uk/R4/CodeSystem/UKCore-NHSNumberVerificationStatus","version":"1.0.0"}]}}],"system":"https://fhir.nhs.uk/Id/nhs-number","value":"5900038181"}],"meta":{"security":[{"code":"U","display":"unrestricted","system":"https://www.hl7.org/fhir/valueset-security-labels.html"}],"versionId":"1"},"name":[{"family":"Bogan","given":["Testing"],"id":"7EB47E61","period":{"start":"2021-01-22"},"prefix":["Mr"],"suffix":["Bsc"],"use":"usual"}],"resourceType":"Patient"}},  # noqa: E231, E501
+    {"scenario":"retrieve_patient","patient":"5900038181", "response":{"address":[{"id":"9F4FF65D","line":["testing","testing","testing","testing","testing"],"period":{"start":"2021-01-22"},"postalCode":"HL12 6HL","use":"home"}],"birthDate":"2010-01-22","extension":[{"extension":[{"url":"language","valueCodeableConcept":{"coding":[{"code":"hy","display":"Armenian","system":"https://fhir.nhs.uk/R4/CodeSystem/UKCore-HumanLanguage","version":"1.0.0"}]}},{"url":"interpreterRequired","valueBoolean":False}],"url":"https://fhir.nhs.uk/R4/StructureDefinition/Extension-UKCore-NHSCommunication"}],"gender":"female","id":"5900038181","identifier":[{"extension":[{"url":"https://fhir.nhs.uk/R4/StructureDefinition/Extension-UKCore-NHSNumberVerificationStatus","valueCodeableConcept":{"coding":[{"code":"01","display":"Number present and verified","system":"https://fhir.nhs.uk/R4/CodeSystem/UKCore-NHSNumberVerificationStatus","version":"1.0.0"}]}}],"system":"https://fhir.nhs.uk/Id/nhs-number","value":"5900038181"}],"meta":{"security":[{"code":"U","display":"unrestricted","system":"https://www.hl7.org/fhir/valueset-security-labels.html"}],"versionId":"19"},"name":[{"family":"Bogan","given":["Testing"],"id":"7EB47E61","period":{"start":"2021-01-22"},"prefix":["Mr"],"suffix":["Bsc"],"use":"usual"}],"resourceType":"Patient"}},  # noqa: E231, E501
```

Comment:
This is more a problem of brittle tests i.e. do we need to have a "deep" compare of records?

Reply:
this change was prior to your update to the patient retrieval test, and was needed to get the PR through the pipelines. I agree though this is a symptom of the brittle tests.